core/domain/user_jobs_one_off.py

```diff
@@ -396,3 +396,22 @@ def map(model_instance):
     @staticmethod
     def reduce(key, values):
         yield (key, len(values))
+
+
+class UserGaeIdOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
I'm not sure why this wasn't caught by the coverage checks, but tests for this job seem to be missing. Please add them before review. 

/cc @lilithxxx

Reply:
The code-climate checks didn't seem to start, I am seeing this for the first time. Maybe push another commit to check if they work correctly?